app/views/org_admin/templates/_funder_templates_list.html.erb

```diff
@@ -0,0 +1,106 @@
+<% # locals: templates, current_org %>
+
+<div class="table-responsive">
+  <table class="table table-hover table-bordered tablesorter">
+    <thead>
+      <% if scope.length > TABLE_FILTER_MIN_ROWS %>
+        <tr>
+          <th colspan="5" class="sorter-false">
+            <%= render(partial: "shared/table_filter", locals: { placeholder: _('Search')}) %>
+          </th>
+        </tr>
+      <% end %>
+      <tr>
+        <th><%= _('Template Name') %></th>
+        <th><%= _('Funder') %></th>
+        <th><%= _('Status') %></th>
+        <th><%= _('Edited Date') %></th>
+        <th class="sorter-false"></th>
+      </tr>
+    </thead>
+    <tbody>
+    <% scope.each do |template| %>
+      <% customization = customizations.select{ |t| t.customization_of == template.dmptemplate_id }.first %>
```

Comment:
Storing customization as a hash with keys being the dmptemplate_id would allow for faster lookup here:)

Reply:
yes. will update that